src/client/io_tracker.cpp

```diff
@@ -184,6 +192,10 @@ void IOTracker::DoWrite(MDSClient* mdsclient, const FInfo_t* fileInfo) {
             break;
     }
 
+    if (throttle) {
+        throttle->Get(false, length_);
```

Comment:
Have you tested performance after throttle here？

Reply:
Tested.
For IOPS throttle, the Get operation can exceed 100,000 qps.
and if disable all throttle, the performance is basically unchanged